packages/@okta/vuepress-site/docs/guides/event-hook-hookdeck/main/index.md

```diff
@@ -0,0 +1,182 @@
+---
+title: Event Hooks with Hookdeck
+excerpt: How to demonstrate Event Hooks using Hookdeck to expose a local app to the internet.
+layout: Guides
+---
+
+<StackSelector />
+
+This guide provides a working example of an Okta Event Hook. It uses the Hookdeck utility to expose a local application to the internet and receive and respond to Event Hook calls. Event Hooks are outbound calls from Okta that can notify your own software systems of events occurring in your Okta org. The Hookdeck utility enables the testing of Event Hooks locally, rather than implementing an internet-based production or test external service.
+
+---
+
+**Learning outcomes**
+
+* Understand the Okta Event Hook calls and responses.
+* Implement Okta Event Hooks using a local application and the Hookdeck utility.
+* Preview and test an Okta Event Hook and review the call details with Hookdeck.
+
+**What you need**
+
+* [Okta Developer Edition organization](https://developer.okta.com/signup/)
+* A local application. See [Create a local application](#create-a-local-application) in this guide to install a sample code application.
+* The Hookdeck utility. See [Install Hookdeck](#install-hookdeck).
+
+**Sample code**
+
+* See [nodejs-webhook-server-example](https://github.com/hookdeck/nodejs-webhook-server-example) for this project's sample code. Installation instructions are available in the next section, [Create a local application](#create-a-local-application).
+
+---
+
+## Create a local application
+
+<StackSnippet snippet="sample-app" />
+
+## Install Hookdeck
+
+If you already have Hookdeck installed, move on to [Run Hookdeck](#run-hookdeck). If not, follow the installation instructions at [Using The Hookdeck CLI](https://hookdeck.com/docs/using-the-cli) to install in your sample application folder (or the location of your choice).
```

Comment:
Using The Hookdeck CLI << should the heading be "Use" to keep all the heading titles consistent